src/main/service/courtService.ts

```diff
@@ -0,0 +1,26 @@
+import { InputFilterService } from './inputFilterService';
+
+export class CourtService {
+  public generateCrownCourtArray(courtsList): object {
```

Comment:
can we split this into a method that creates a 2 dimensional array for alphabet and courts and one that limits it to the crown court? that way we can reuse this method on this journey and the one on listings for the other alphabet page